api/routes/auth.py

```diff
@@ -29,13 +32,25 @@ def build_protected_response(url: str, token: str):
 
 
 @router.get("/login", status_code=status.HTTP_307_TEMPORARY_REDIRECT)
-async def login():
-    params = {"client_id": GITHUB_CLIENT_ID}
+async def login(
+    referrer: str | None = None,
```

Comment:
I am curious if there is a way to pass this in as a header?

Reply:
I am not sure actually. The login button is an `<a>` tag, so I am not sure.